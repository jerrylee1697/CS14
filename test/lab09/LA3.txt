92 105
Alhambra
Anaheim_Canyon
Arleta
Avocado_Hts
Azuza
Barstow
Beaumont
Bellflower
Buena_Park
Burbank
Carson
Chino
Colton
Commerce
Compton
Corona
Costa_Mesa
Devore
Diamond_Bar
Downey
Duarte
EastLA
East_LA_Xchange
East_Orange
El_Monte
El_Segundo
El_Toro
Encino
Fontana
Four_Level
Fullerton
Gardena
Glendale
Glendora
Granada_Hills
Hawthorne
Irvine
Irvine_Meadows
Irwindale
La_Canada
Ladera_Hts
Laguna
Lancaster
Lincoln_Heights
Long_Beach
Los_Alamitos
Marina_Del_Rey
Mira_Loma
Mission_Viejo
Montecito
Moorpark
Moreno_Valley
Murrieta
Newhall
Newport_Beach
Norwalk
Ontario
Orange
Pacoima
Pasadena
Pico-Union
Placentia
Pomona
Prado
Rancho_Cucamonga
Rancho_Santa_Margarita
Redlands
Riverside
San_Bernardino
San_Clemente
San_Fernando
San_Pedro
Santa_Anna
Santa_Clarita
Santa_Fe_Springs
Santa_Monica
Signal_Hill
Silver_Lake
Studio_City
Sun_Valley
Sylmar
Temecula
Thousand_Oaks
Toll_Plaza
Tustin
Van_Nuys
Ventura
Victorville
Watts
WestLA
Westminster
Yorba_Linda
GoldenState_5-1 7 Santa_Clarita Newhall
GoldenState_5-4 2 San_Fernando Arleta
GoldenState_5-6 11 Sun_Valley Burbank
GoldenState_5-8 2 Silver_Lake Montecito
GoldenState_5-9 2 Montecito Lincoln_Heights
Downtown_LA_Tangle 2 Lincoln_Heights East_LA_Xchange
SantaAnna_5-2 7 Commerce Santa_Fe_Springs
SantaAnna_5-4 8 Buena_Park Orange
SantaAnna_5-5 12 Orange Santa_Anna
SantaAnna_5-6 2 Santa_Anna Tustin
SantaAnna_5-8 4 Irvine El_Toro
SanDiego_405-2 7.5 Granada_Hills Encino
SanDiego_405-3 10 Encino WestLA
SanDiego_405-4 3 WestLA Ladera_Hts
SanDiego_405-5 5 Ladera_Hts Hawthorne
SanDiego_405-6 8 Hawthorne Carson
SanDiego_405-7 5.5 Carson Signal_Hill
SanDiego_405-8 8 Signal_Hill Los_Alamitos
SanDiego_405-9 2 Los_Alamitos Westminster
SanDiego_405-11 8 Costa_Mesa Irvine_Meadows
SanDiego_5-1 10 El_Toro Mission_Viejo
SanDiego_5-2 11 Mission_Viejo San_Clemente
Corona_Del_Mar_73-1 10 Costa_Mesa Laguna
Corona_Del_Mar_73-2 6.75 Laguna Mission_Viejo
Antelope_14 43 Newhall Lancaster
Glendale_2-2 4 Glendale Silver_Lake
Moorpark_23 6 Thousand_Oaks Moorpark
Reagan_118-1 24 Moorpark Granada_Hills
Reagan_118-3 2 Arleta Pacoima
Ventura_101-2 24 Thousand_Oaks Van_Nuys
Ventura_101-3 6 Van_Nuys Studio_City
Ventura_134-1 5 Studio_City Burbank
Ventura_134-3 4.5 Glendale Pasadena
Hollywood_170 9 Sun_Valley Studio_City
Hollywood_101-1 10 Studio_City Four_Level
Hollywood_101-2 3 Four_Level Lincoln_Heights
Foothill_210-1 6 Sylmar Pacoima
Foothill_210-3 5 La_Canada Pasadena
Foothill_210-4 11.4 Pasadena Irwindale
Foothill_210-5 8.5 Irwindale Glendora
Foothill_210-6 18 Glendora Rancho_Cucamonga
Foothill_210-7 10.5 Rancho_Cucamonga San_Bernardino
Mojave_15 27 Victorville Devore
Ontario_15-2 5.5 Rancho_Cucamonga Ontario
Ontario_15-3 3.2 Ontario Mira_Loma
Ontario_15-4 10 Ontario Corona
Temecula_15-1 10 Corona Murrieta
Temecula_15-2 22 Murrieta Temecula
Barstow_215-2 6 San_Bernardino Colton
Riverside_215 6 Colton Riverside
Riverside_91-1 14.2 Riverside Corona
Riverside_91-3 5 Prado Anaheim_Canyon
Riverside_91-5 3 Yorba_Linda Placentia
Riverside_91-6 2.6 Placentia Fullerton
Riverside_91-7 6 Fullerton Bellflower
Artesia_91 5.4 Bellflower Compton
Gardena_91 5.3 Compton Gardena
GardenGrove_22-2 2.7 Orange East_Orange
SanBernardino_10-1 15.6 Beaumont Redlands
SanBernardino_10-2 5.3 Redlands Colton
SanBernardino_10-4 15.5 Ontario Pomona
SanBernardino_10-5 11.5 Pomona El_Monte
SanBernardino_10-6 19.7 El_Monte Alhambra
SanBernardino_10-7 1.5 Alhambra Lincoln_Heights
SantaMonica_10-1 5 Santa_Monica WestLA
SantaMonica_10-3 9.7 Pico-Union East_LA_Xchange
Pomona_60-1 3.3 East_LA_Xchange EastLA
Pomona_60-2 8.4 EastLA Avocado_Hts
Pomona_60-3 11.8 Avocado_Hts Diamond_Bar
Pomona_60-4 5.7 Diamond_Bar Chino
Pomona_60-5 10.1 Chino Mira_Loma
Pomona_60-6 11.6 Mira_Loma Riverside
MoVal_60-2 18.3 Moreno_Valley Beaumont
Escondido_215 28.8 Moreno_Valley Murrieta
ChinoVal_71-1 3.3 Pomona Chino
Orange_57-1 4.6 Glendora Pomona
Orange_57-3 9.1 Diamond_Bar Placentia
Orange_57-4 4.8 Placentia Orange
CostaMesa_55-1 4.9 Yorba_Linda East_Orange
CostaMesa_55-3 3.9 Santa_Anna Costa_Mesa
EasternToll_241-1 8 Anaheim_Canyon Toll_Plaza
FoothillToll_241-2 17 Toll_Plaza Rancho_Santa_Margarita
EasternToll_261 4 Toll_Plaza Tustin
Laguna_133-1 6 Irvine Irvine_Meadows
Laguna_133-2 1.2 Irvine_Meadows Laguna
SanGabriel_605-1 5.6 Duarte El_Monte
SanGabriel_605-2 3.8 El_Monte Avocado_Hts
SanGabriel_605-3 7.8 Avocado_Hts Santa_Fe_Springs
SanGabriel_605-4 2 Santa_Fe_Springs Norwalk
SanGabriel_605-5 2.6 Norwalk Bellflower
SanGabriel_605-6 5.2 Bellflower Westminster
LongBeach_710-1 1.8 Alhambra EastLA
LongBeach_710-2 1.2 EastLA Commerce
LongBeach_710-3 7.6 Commerce Downey
LongBeach_710-4 2.7 Downey Compton
LongBeach_710-5 3.5 Compton Signal_Hill
SeaSide_47 3.5 San_Pedro Long_Beach
Harbor_110-1 8 San_Pedro Carson
Harbor_110-2 1.9 Carson Gardena
Harbor_110-4 7.4 Watts Pico-Union
Harbor_110-5 12 Pico-Union Four_Level
Arroyo_110-1 2.3 Four_Level Montecito
Arroyo_110-2 8 Montecito Pasadena
Century-105-1 4.3 Norwalk Downey
Century-105-3 5.2 Watts Hawthorne