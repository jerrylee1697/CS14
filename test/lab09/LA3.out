Algorithm: NUTs 
Weight of the tree: 682.85
Edges in the tree: Antelope_14 Artesia_91 Barstow_215-2 Corona_Del_Mar_73-1 Corona_Del_Mar_73-2 CostaMesa_55-3 Downtown_LA_Tangle EasternToll_241-1 EasternToll_261 FoothillToll_241-2 Foothill_210-1 Foothill_210-3 Foothill_210-4 Foothill_210-5 Foothill_210-6 Foothill_210-7 GardenGrove_22-2 Gardena_91 Glendale_2-2 GoldenState_5-1 GoldenState_5-4 GoldenState_5-6 GoldenState_5-8 GoldenState_5-9 Harbor_110-1 Harbor_110-4 Hollywood_101-1 Hollywood_101-2 LongBeach_710-3 MoVal_60-2 Mojave_15 Moorpark_23 Ontario_15-2 Ontario_15-3 Ontario_15-4 Orange_57-3 Orange_57-4 Pomona_60-1 Pomona_60-2 Pomona_60-3 Pomona_60-4 Reagan_118-1 Reagan_118-3 Riverside_215 Riverside_91-3 Riverside_91-5 Riverside_91-6 Riverside_91-7 SanBernardino_10-1 SanBernardino_10-2 SanBernardino_10-4 SanBernardino_10-5 SanBernardino_10-6 SanDiego_405-11 SanDiego_405-2 SanDiego_405-3 SanDiego_405-4 SanDiego_405-5 SanDiego_405-6 SanDiego_405-7 SanDiego_405-8 SanDiego_405-9 SanDiego_5-1 SanDiego_5-2 SanGabriel_605-1 SanGabriel_605-3 SanGabriel_605-4 SantaAnna_5-2 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 SantaAnna_5-8 SantaMonica_10-1 SantaMonica_10-3 SeaSide_47 Temecula_15-1 Temecula_15-2 Ventura_101-2 Ventura_101-3 Ventura_134-1 Ventura_134-3 Number of fragments in tree: 4
Fragment 1 contains vertex Alhambra and the following 75 edges: GoldenState_5-6 Ventura_134-1 Ventura_101-3 Ventura_101-2 Moorpark_23 Reagan_118-1 SanDiego_405-2 SanDiego_405-3 SanDiego_405-4 SanDiego_405-5 SanDiego_405-6 SanDiego_405-7 SanDiego_405-8 SanDiego_405-9 Harbor_110-1 SeaSide_47 SantaMonica_10-1 Hollywood_101-1 Hollywood_101-2 GoldenState_5-9 GoldenState_5-8 Glendale_2-2 Ventura_134-3 Foothill_210-3 Foothill_210-4 Foothill_210-5 Foothill_210-6 Foothill_210-7 Barstow_215-2 Riverside_215 SanBernardino_10-2 SanBernardino_10-1 MoVal_60-2 Ontario_15-2 Ontario_15-3 Ontario_15-4 Temecula_15-1 Temecula_15-2 SanBernardino_10-4 SanBernardino_10-5 SanBernardino_10-6 SanGabriel_605-1 Downtown_LA_Tangle SantaMonica_10-3 Harbor_110-4 Pomona_60-1 Pomona_60-2 Pomona_60-3 Pomona_60-4 Orange_57-3 Riverside_91-5 Riverside_91-6 Riverside_91-7 Artesia_91 Gardena_91 Orange_57-4 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 EasternToll_261 EasternToll_241-1 Riverside_91-3 FoothillToll_241-2 CostaMesa_55-3 SanDiego_405-11 Corona_Del_Mar_73-1 Corona_Del_Mar_73-2 SanDiego_5-1 SantaAnna_5-8 SanDiego_5-2 GardenGrove_22-2 SanGabriel_605-3 SantaAnna_5-2 LongBeach_710-3 SanGabriel_605-4 
Fragment 2 contains vertex Arleta and the following 3 edges: GoldenState_5-4 Reagan_118-3 Foothill_210-1 
Fragment 3 contains vertex Lancaster and the following 2 edges: GoldenState_5-1 Antelope_14 
Fragment 4 contains vertex Devore and the following 1 edges: Mojave_15 
Algorithm: Kruskal's 
Weight of the tree: 559.25
Edges in the tree: Antelope_14 Arroyo_110-1 Barstow_215-2 Century-105-1 Century-105-3 ChinoVal_71-1 Corona_Del_Mar_73-2 CostaMesa_55-3 Downtown_LA_Tangle EasternToll_241-1 EasternToll_261 FoothillToll_241-2 Foothill_210-1 Foothill_210-3 Foothill_210-5 Foothill_210-7 GardenGrove_22-2 Gardena_91 Glendale_2-2 GoldenState_5-1 GoldenState_5-4 GoldenState_5-8 GoldenState_5-9 Harbor_110-1 Harbor_110-2 Harbor_110-4 Hollywood_101-1 Hollywood_170 Laguna_133-1 Laguna_133-2 LongBeach_710-1 LongBeach_710-2 LongBeach_710-4 LongBeach_710-5 MoVal_60-2 Mojave_15 Moorpark_23 Ontario_15-2 Ontario_15-3 Ontario_15-4 Orange_57-1 Orange_57-3 Orange_57-4 Pomona_60-4 Pomona_60-5 Reagan_118-3 Riverside_215 Riverside_91-3 Riverside_91-5 Riverside_91-6 Riverside_91-7 SanBernardino_10-1 SanBernardino_10-2 SanBernardino_10-7 SanDiego_405-11 SanDiego_405-2 SanDiego_405-3 SanDiego_405-4 SanDiego_405-5 SanDiego_405-6 SanDiego_405-9 SanDiego_5-2 SanGabriel_605-1 SanGabriel_605-2 SanGabriel_605-3 SanGabriel_605-4 SanGabriel_605-5 SanGabriel_605-6 SantaAnna_5-2 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 SantaAnna_5-8 SantaMonica_10-1 SeaSide_47 Temecula_15-1 Temecula_15-2 Ventura_101-2 Ventura_101-3 Ventura_134-1 Ventura_134-3 Number of fragments in tree: 4
Fragment 1 contains vertex Alhambra and the following 75 edges: GoldenState_5-8 Glendale_2-2 Ventura_134-3 Foothill_210-3 GoldenState_5-9 Downtown_LA_Tangle SanBernardino_10-7 LongBeach_710-1 LongBeach_710-2 SantaAnna_5-2 SanGabriel_605-3 SanGabriel_605-2 SanGabriel_605-1 SanGabriel_605-4 SanGabriel_605-5 Riverside_91-7 Riverside_91-6 Riverside_91-5 Orange_57-3 Pomona_60-4 Pomona_60-5 Ontario_15-3 Ontario_15-2 Foothill_210-7 Barstow_215-2 Riverside_215 SanBernardino_10-2 SanBernardino_10-1 MoVal_60-2 Ontario_15-4 Temecula_15-1 Temecula_15-2 ChinoVal_71-1 Orange_57-1 Foothill_210-5 Orange_57-4 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 EasternToll_261 EasternToll_241-1 Riverside_91-3 FoothillToll_241-2 CostaMesa_55-3 SanDiego_405-11 Laguna_133-1 SantaAnna_5-8 Laguna_133-2 Corona_Del_Mar_73-2 SanDiego_5-2 GardenGrove_22-2 SanGabriel_605-6 SanDiego_405-9 Century-105-1 LongBeach_710-4 Gardena_91 Harbor_110-2 SanDiego_405-6 SanDiego_405-5 SanDiego_405-4 SanDiego_405-3 SanDiego_405-2 SantaMonica_10-1 Century-105-3 Harbor_110-4 Harbor_110-1 SeaSide_47 LongBeach_710-5 Arroyo_110-1 Hollywood_101-1 Ventura_101-3 Ventura_101-2 Moorpark_23 Ventura_134-1 Hollywood_170 
Fragment 2 contains vertex Arleta and the following 3 edges: GoldenState_5-4 Reagan_118-3 Foothill_210-1 
Fragment 3 contains vertex Lancaster and the following 2 edges: GoldenState_5-1 Antelope_14 
Fragment 4 contains vertex Devore and the following 1 edges: Mojave_15 
Algorithm: Prim's 
Weight of the tree: 559.25
Edges in the tree: Antelope_14 Arroyo_110-1 Barstow_215-2 Century-105-1 Century-105-3 ChinoVal_71-1 Corona_Del_Mar_73-2 CostaMesa_55-3 Downtown_LA_Tangle EasternToll_241-1 EasternToll_261 FoothillToll_241-2 Foothill_210-1 Foothill_210-3 Foothill_210-5 Foothill_210-7 GardenGrove_22-2 Gardena_91 Glendale_2-2 GoldenState_5-1 GoldenState_5-4 GoldenState_5-8 GoldenState_5-9 Harbor_110-1 Harbor_110-2 Harbor_110-4 Hollywood_101-1 Hollywood_170 Laguna_133-1 Laguna_133-2 LongBeach_710-1 LongBeach_710-2 LongBeach_710-4 LongBeach_710-5 MoVal_60-2 Mojave_15 Moorpark_23 Ontario_15-2 Ontario_15-3 Ontario_15-4 Orange_57-1 Orange_57-3 Orange_57-4 Pomona_60-4 Pomona_60-5 Reagan_118-1 Reagan_118-3 Riverside_215 Riverside_91-3 Riverside_91-5 Riverside_91-6 Riverside_91-7 SanBernardino_10-1 SanBernardino_10-2 SanBernardino_10-7 SanDiego_405-11 SanDiego_405-2 SanDiego_405-3 SanDiego_405-4 SanDiego_405-5 SanDiego_405-6 SanDiego_405-9 SanDiego_5-2 SanGabriel_605-1 SanGabriel_605-2 SanGabriel_605-3 SanGabriel_605-4 SanGabriel_605-5 SanGabriel_605-6 SantaAnna_5-2 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 SantaAnna_5-8 SantaMonica_10-1 SeaSide_47 Temecula_15-1 Temecula_15-2 Ventura_101-3 Ventura_134-1 Ventura_134-3 Number of fragments in tree: 4
Fragment 1 contains vertex Alhambra and the following 75 edges: GoldenState_5-8 Glendale_2-2 Ventura_134-3 Foothill_210-3 GoldenState_5-9 Downtown_LA_Tangle SanBernardino_10-7 LongBeach_710-1 LongBeach_710-2 SantaAnna_5-2 SanGabriel_605-3 SanGabriel_605-2 SanGabriel_605-1 SanGabriel_605-4 SanGabriel_605-5 Riverside_91-7 Riverside_91-6 Riverside_91-5 Orange_57-3 Pomona_60-4 Pomona_60-5 Ontario_15-3 Ontario_15-2 Foothill_210-7 Barstow_215-2 Riverside_215 SanBernardino_10-2 SanBernardino_10-1 MoVal_60-2 Ontario_15-4 Temecula_15-1 Temecula_15-2 ChinoVal_71-1 Orange_57-1 Foothill_210-5 Orange_57-4 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 EasternToll_261 EasternToll_241-1 Riverside_91-3 FoothillToll_241-2 CostaMesa_55-3 SanDiego_405-11 Laguna_133-1 SantaAnna_5-8 Laguna_133-2 Corona_Del_Mar_73-2 SanDiego_5-2 GardenGrove_22-2 SanGabriel_605-6 SanDiego_405-9 Century-105-1 LongBeach_710-4 Gardena_91 Harbor_110-2 SanDiego_405-6 SanDiego_405-5 SanDiego_405-4 SanDiego_405-3 SanDiego_405-2 Reagan_118-1 Moorpark_23 SantaMonica_10-1 Century-105-3 Harbor_110-4 Harbor_110-1 SeaSide_47 LongBeach_710-5 Arroyo_110-1 Hollywood_101-1 Ventura_101-3 Ventura_134-1 Hollywood_170 
Fragment 2 contains vertex Arleta and the following 3 edges: GoldenState_5-4 Reagan_118-3 Foothill_210-1 
Fragment 3 contains vertex Lancaster and the following 2 edges: GoldenState_5-1 Antelope_14 
Fragment 4 contains vertex Devore and the following 1 edges: Mojave_15 
Algorithm: YAMA 
Weight of the tree: 559.25
Edges in the tree: Antelope_14 Arroyo_110-1 Barstow_215-2 Century-105-1 Century-105-3 ChinoVal_71-1 Corona_Del_Mar_73-2 CostaMesa_55-3 Downtown_LA_Tangle EasternToll_241-1 EasternToll_261 FoothillToll_241-2 Foothill_210-1 Foothill_210-3 Foothill_210-5 Foothill_210-7 GardenGrove_22-2 Gardena_91 Glendale_2-2 GoldenState_5-1 GoldenState_5-4 GoldenState_5-8 GoldenState_5-9 Harbor_110-1 Harbor_110-2 Harbor_110-4 Hollywood_101-1 Hollywood_170 Laguna_133-1 Laguna_133-2 LongBeach_710-1 LongBeach_710-2 LongBeach_710-4 LongBeach_710-5 MoVal_60-2 Mojave_15 Moorpark_23 Ontario_15-2 Ontario_15-3 Ontario_15-4 Orange_57-1 Orange_57-3 Orange_57-4 Pomona_60-4 Pomona_60-5 Reagan_118-1 Reagan_118-3 Riverside_215 Riverside_91-3 Riverside_91-5 Riverside_91-6 Riverside_91-7 SanBernardino_10-1 SanBernardino_10-2 SanBernardino_10-7 SanDiego_405-11 SanDiego_405-2 SanDiego_405-3 SanDiego_405-4 SanDiego_405-5 SanDiego_405-6 SanDiego_405-9 SanDiego_5-2 SanGabriel_605-1 SanGabriel_605-2 SanGabriel_605-3 SanGabriel_605-4 SanGabriel_605-5 SanGabriel_605-6 SantaAnna_5-2 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 SantaAnna_5-8 SantaMonica_10-1 SeaSide_47 Temecula_15-1 Temecula_15-2 Ventura_101-3 Ventura_134-1 Ventura_134-3 Number of fragments in tree: 4
Fragment 1 contains vertex Alhambra and the following 75 edges: GoldenState_5-8 Glendale_2-2 Ventura_134-3 Foothill_210-3 GoldenState_5-9 Downtown_LA_Tangle SanBernardino_10-7 LongBeach_710-1 LongBeach_710-2 SantaAnna_5-2 SanGabriel_605-3 SanGabriel_605-2 SanGabriel_605-1 SanGabriel_605-4 SanGabriel_605-5 Riverside_91-7 Riverside_91-6 Riverside_91-5 Orange_57-3 Pomona_60-4 Pomona_60-5 Ontario_15-3 Ontario_15-2 Foothill_210-7 Barstow_215-2 Riverside_215 SanBernardino_10-2 SanBernardino_10-1 MoVal_60-2 Ontario_15-4 Temecula_15-1 Temecula_15-2 ChinoVal_71-1 Orange_57-1 Foothill_210-5 Orange_57-4 SantaAnna_5-4 SantaAnna_5-5 SantaAnna_5-6 EasternToll_261 EasternToll_241-1 Riverside_91-3 FoothillToll_241-2 CostaMesa_55-3 SanDiego_405-11 Laguna_133-1 SantaAnna_5-8 Laguna_133-2 Corona_Del_Mar_73-2 SanDiego_5-2 GardenGrove_22-2 SanGabriel_605-6 SanDiego_405-9 Century-105-1 LongBeach_710-4 Gardena_91 Harbor_110-2 SanDiego_405-6 SanDiego_405-5 SanDiego_405-4 SanDiego_405-3 SanDiego_405-2 Reagan_118-1 Moorpark_23 SantaMonica_10-1 Century-105-3 Harbor_110-4 Harbor_110-1 SeaSide_47 LongBeach_710-5 Arroyo_110-1 Hollywood_101-1 Ventura_101-3 Ventura_134-1 Hollywood_170 
Fragment 2 contains vertex Arleta and the following 3 edges: GoldenState_5-4 Reagan_118-3 Foothill_210-1 
Fragment 3 contains vertex Lancaster and the following 2 edges: GoldenState_5-1 Antelope_14 
Fragment 4 contains vertex Devore and the following 1 edges: Mojave_15 
